docs/diagrams/LogicClassDiagram.puml

```diff
@@ -4,31 +4,28 @@ skinparam arrowThickness 1.1
 skinparam arrowColor LOGIC_COLOR_T4
 skinparam classBackgroundColor LOGIC_COLOR
 
-package Logic {
-
-Class AddressBookParser
+Package Logic as LogicPackage {
+Class SchedulerParser
 Class XYZCommand
 Class CommandResult
 Class "{abstract}\nCommand" as Command
 
-
 Class "<<interface>>\nLogic" as Logic
 Class LogicManager
 }
 
-package Model{
+Package Model{
```

Comment:
will changing the capital letter affect anything?

Reply:
Nope. Intended for standardization

Reply:
Will be converting back to small caps, as it seems to be the more frequently used.